pxr/usd/usd/testenv/testUsdSchemaRegistry/resources/schema.usda

```diff
@@ -35,6 +35,7 @@ class "AbstractTest" (
 
 class MetadataTest "MetadataTest" (
     inherits = </AbstractTest>
+    displayName = "Display Name"
```

Comment:
I don't think we should allow `displayName` to be authored into schema definitions - it's too much of a landmine, that your **type** can effectively override the name you've given to a prim!  Accordingly, `usdGenSchema` should detect and flag this as an invalid schema definition, which we should also test in `testUsdSchemaGen` .

Reply:
Removed